app/uk/gov/hmrc/claimvatenrolmentfrontend/featureswitch/frontend/views/feature_switch.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.claimvatenrolmentfrontend.config.AppConfig
+@import uk.gov.hmrc.claimvatenrolmentfrontend.featureswitch.core.models.FeatureSwitchSetting
+@import uk.gov.hmrc.claimvatenrolmentfrontend.featureswitch.frontend.models.FeatureSwitchProvider
+@import uk.gov.hmrc.claimvatenrolmentfrontend.views.html.templates._
+
+@this(layout: layout,
+        govukFieldset: GovukFieldset,
+        govukCheckboxes: GovukCheckboxes,
+        govukButton: GovukButton,
+        formWithCSRF: FormWithCSRF
+)
+
+@(featureSwitchList: Seq[(FeatureSwitchProvider, Seq[FeatureSwitchSetting])], formAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(Some(titleNoForm("Choose which features to enable."))) {
+<h1 class="govuk-heading-l" >Choose which features to enable.</h1>
+
+    @formWithCSRF(action = formAction) {
+        @for(featureSwitches <- featureSwitchList) {
+            @govukCheckboxes(Checkboxes(
+                fieldset = Some(Fieldset(
+                    legend = Some(Legend(
+                        content = Text(featureSwitches._1.appName),
+                        classes = "govuk-fieldset__legend--m",
+                        isPageHeading = false
+                    ))
+                )),
```

Comment:
Because we don't have the FE/BE split like the other GRS services we could just have one title. So we don't need '<h1 class="govuk-heading-l" >Choose which features to enable.</h1>' above and in the fieldset we could have the content as Text("Choose which feature switches to enable"), classes = "govuk-fieldset__legend--l" and isPageHeading = true